widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -260,9 +269,14 @@ export function Quote(props: QuoteProps) {
       };
     });
   };
-  const steps = getQuoteSteps(quote.result?.swaps ?? []);
+  const steps = getQuoteSteps(selectedQuote?.swaps ?? []);
+
   const numberOfSteps = steps.length;
-  const tooltipContainer = getContainer();
+  const container = getContainer();
+  const sortedQuoteTags = sortTags(props.quote.tags || []);
+  const isQuoteSelected = selectedQuote?.requestId === props.quote.requestId;
+
```

Comment:
I think just 'selected' is good for this variable name.

Reply:
fixed